src/azul/service/config/request_config.json

```diff
@@ -5,16 +5,16 @@
     "specimenId": "bundles.contents.specimens.biomaterial_id",
     "organ": "bundles.contents.specimens.organ",
     "organPart": "bundles.contents.specimens.organ_part",
-    "libraryConstructionApproach": "bundles.contents.processes.library_construction",
-    "genusSpecies": "bundles.contents.specimens.species",
-    "organismAge": "bundles.contents.specimens.age",
-    "organismAgeUnit": "bundles.contents.specimens.age_unit",
-    "biologicalSex": "bundles.contents.specimens.sex",
+    "libraryConstructionApproach": "bundles.contents.processes.library_construction_approach",
+    "genusSpecies": "bundles.contents.specimens.genus_species",
+    "organismAge": "bundles.contents.specimens.organism_age",
+    "organismAgeUnit": "bundles.contents.specimens.organism_age_unit",
+    "biologicalSex": "bundles.contents.specimens.biological_sex",
     "disease": "bundles.contents.specimens.disease",
     "instrumentManufacturerModel": "bundles.contents.processes.instrument",
-    "project": "bundles.contents.project.project",
+    "project": "bundles.contents.project.project_shortname",
```

Comment:
These renames should have broken your new tests, @alex-hancock, or required changes to the assertions.

One of the tests, I think the `…correctness` one, should assert the index contents verbatim, field by field.